start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
+        echo
+        echo "Error!"
+        echo "The Triton CLI tool does not appear to be installed!"
+        echo
+        echo "Please visit:"
+        echo "https://www.joyent.com/blog/introducing-the-triton-command-line-tool"
+        echo "for installation instructions."
+        echo
+
+        sleep 3
+        command -v open >/dev/null 2>&1 && `open https://www.joyent.com/blog/introducing-the-triton-command-line-tool` || true
+
+        exit 1
+    fi
+
+    # Get username from Docker
+    local docker_user=$(docker info 2>&1 | grep "SDCAccount:" | awk -F": " '{print $2}' OFS="/")
+
+    # Get username from Triton
+    local triton_user=$(triton profile get | grep "account:" | awk -F": " '{print $2}' OFS="/")
+
+    # Get DC from Docker
+    local docker_dc=$(echo $DOCKER_HOST | awk -F"/" '{print $3}' OFS="/" | awk -F"\." '{print $1}' OFS="/")
+
+    # Get DC from Triton
+    local triton_dc=$(triton profile get | grep "url:" | awk -F"/" '{print $3}' OFS="/" | awk -F"\." '{print $1}' OFS="/")
+
+    if [ ! "$docker_user" = "$triton_user" ] || [ ! "$docker_dc" = "$triton_dc" ]; then
+        echo
+        echo "Error!"
+        echo "The Triton CLI configuration does not match the Docker CLI configuration!"
+        echo
+        echo "Docker user: ${docker_user}"
+        echo "Triton user: ${docker_user}"
+        echo
+        echo "Docker data center: ${docker_dc}"
+        echo "Triton data center: ${triton_dc}"
+        echo
+        echo "The Triton CLI tool must be configured to use the same user and data center as the Docker CLI."
+        echo
+        echo "Please visit:"
+        echo "https://www.joyent.com/blog/introducing-the-triton-command-line-tool#using-profiles"
+        echo "for instructions on how to configure and set profiles for Triton."
+        echo
+
+        sleep 3
+        command -v open >/dev/null 2>&1 && `open "https://www.joyent.com/blog/introducing-the-triton-command-line-tool#using-profiles"` || true
+
+        exit 1
```

Comment:
I may be reading this wrong but I think this means that we're now *requiring* this blueprint be deployed on Triton? Your Docker client can't be pointed at a local docker machine with this section in place. I'm fine with doing that, but if so then we should be explicit about it and remove things the `docker-compose-local.yml`, remove the chunks of code to support docker-machine, update the README, etc. I'd also suggest that if we're going to do that then we should consider going whole-hog and requiring Triton CNS as well so that the demo is complete.